draft-ietf-quic-transport.md

```diff
@@ -2313,6 +2304,13 @@ the application requests that the connection be closed.  The application
 protocol can use an APPLICATION_CLOSE message with an appropriate error code to
 signal closure.
 
+If the connection has been successfully established, endpoints MUST send any
+closing frames in a 1-RTT packet.  Prior to connection establishment, endpoints
+SHOULD send closing frames in a Handshake packet.  An endpoint MAY send closing
```

Comment:
Does this need a caveat like "...SHOULD send closing frames in a Handshake packet if possible."